content/open_science/initiatives/open_science.md

```diff
@@ -0,0 +1,70 @@
+---
```

Comment:
Can't the file name be `our_initiatives.md` ?
To keep the URL consistent with the breadcrumbs